app/src/main/java/com/heapixLearn/discovery/logic/contact/contacts_domain_interfaces/DBStoreContactManager.java

```diff
@@ -0,0 +1,16 @@
+package com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces;
+
+import com.heapixLearn.discovery.logic.contact.LogicContact;
+
+import java.util.ArrayList;
+
+public interface DBStoreContactManager {
+    Void create(LogicContact data);
+    Void update(LogicContact data);
+    Void delete(LogicContact data);
+    LogicContact getById(LogicContact data);
+    ArrayList<LogicContact> getAll();
+    void onContactCreated(LogicContact contact);
+    void omContactDeleted(LogicContact contact);
```

Comment:
@dianaprimshits typo, supposed to be `onContactDeleted`

Reply:
@YodaMike This unnecessary method has been removed.